CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ All notable changes to this project will be documented in this file. See [standa
 
 ### Supported CDK Version
 
-* [2.54.0](https://github.com/aws/aws-cdk/releases/tag/v2.54.0)
+* [2.69.0](https://github.com/aws/aws-cdk/releases/tag/v2.69.0)
```

Comment:
This shouldn't be changed. It's automatically generated when we do the release.

Reply:
The change has now been removed!